Foundation/NSLog.swift

```diff
@@ -41,3 +41,11 @@ public func NSLog(_ format: String, _ args: CVarArg...) {
         NSLogv(format, $0) 
     }
 }
+
+// hook for System.out.println()
+#if os(Android)
+public var ANLog = {
```

Comment:
It’s used for debugging. It’s replaced by a closure calling System.out.println() by the app.